Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml

```diff
@@ -0,0 +1,89 @@
MywxLjUzLTYuNjQsMi4zMy0xMEExLjI2LDEuMjYsMCwwLDAsNDUuMjYsMjcuMDlaIi8+PHBhdGggZD0iTTQ5LjQ5LDI1Yy0uMDUtLjE5LS4yNS0uMzItLjM4LS40NmE3LDcsMCwwLDAtLjgzLS43Yy0uNDMtLjMzLS44Ny0uNjUtMS4zMi0uOTQtLjktLjU3LTEuNzktMS4xNC0yLjcyLTEuNjUtMS40NC0uNzktMy0xLjM3LTQuNDctMmExLjY4LDEuNjgsMCwwLDAtMiwuMzJMMzUuNSwyMS42N2E0LjQ0LDQuNDQsMCwwLDAtLjMzLjM3Yy45NC4zMiwxLjgzLjYxLDIuNzEuOTNsNS4zMywyYy4zMS4xMiwyLC42LDIuMTYuNjVhNC44OSw0Ljg5LDAsMCwwLDIuNTQuMTcsOS45Miw5LjkyLDAsMCwwLDEuMzItLjNBLjQ0LjQ0LDAsMCwwLDQ5LjQ5LDI1WiIvPjxwYXRoIGQ9Ik0zMi45NCwyMy4wNywzMS42OSwzNC43MWwtMTUtNC4yM3YtLjIxWiIvPjxwYXRoIGQ9Ik0zMi42NCwyMi40M2MtNS42OSwyLjQ4LTExLjE5LDUtMTYuNzUsNy40Mi0uMzYsMC0uNC0uMjEtLjMxLS41NS43Ni0yLjc4LDEuNTEtNS41OCwyLjI5LTguMzZhMy40MiwzLjQyLDAsMCwxLC42Mi0xLjM3LDMuMzksMy4zOSwwLDAsMSwzLjgzLS43NmMzLjEsMS41Miw2LjQ4LDIuMjEsOS43LDMuNFoiLz48cGF0aCBkPSJNNDMuOTMsMzguODJjLjIxLS45NC40Ny0xLjkzLjY3LTIuNzkuNjMtMi42OCwxLjI1LTUuMzYsMS45LTgsLjA2LS4yNi4yNi0uNjYuNDMtLjY4YTEuMTcsMS4xNywwLDAsMSwuODUuMzRRNTEuNDEsMzEuMjUsNTUsMzQuOWExLjIsMS4yLDAsMCwxLC4zLjczYzAsMS4xNSwwLDMuNTMsMCwzLjUzYTE3LDE3LDAsMCwwLTIuNTYtLjY4LDE5Ljg1LDE5Ljg1LDAsMCwwLTMuNy0uMjcsMzIuOTIsMzIuOTIsMCwwLDAtMy43MS4yOUE4LjM5LDguMzksMCwwLDAsNDMuOTMsMzguODJaIi8+PHBhdGggZD0iTTIwLjkzLDQxLjQ1bDEwLjY4LTUuODhjMCwuMzcuMDUuNjQuMDcuOTEuMTUsMi44Mi4yNSw2LjY5LjUyLDlhNCw0LDAsMCwxLS42NiwyLjU1LDUuNzEsNS43MSwwLDAsMS0uNjUuNyw5LjA5LDkuMDksMCwwLDEtLjgtLjUyTDIxLjQsNDEuODNDMjEuMjYsNDEuNzMsMjEuMTMsNDEuNjEsMjAuOTMsNDEuNDVaIi8+PHBhdGggZD0iTTU2LDM0LjlsLTcuNTctNy42MWMtLjItLjIxLS41NS0uNDEtLjI5LS43NGExLjEzLDEuMTMsMCwwLDEsLjY4LS4zMSw3LjMxLDcuMzEsMCwwLDEsMy40NS4wNywxMS42MSwxMS42MSwwLDAsMSw3LjMzLDcuMjMsMTEuNzUsMTEuNzUsMCwwLDEsMCw3LjIzYy0uMTYuNDgtLjY0LDEuNjctLjY0LDEuNjdzLS41MS0uMzgtLjYtLjQ3Yy0uNzEtLjY2LTIuMzktMi4zNS0yLjM5LTIuMzVaIi8+PHBhdGggZD0iTTE2Ljg2LDU0LjA3LDE4LjEzLDQ4YzEuMzEuODksMi41NSwxLjcyLDMuNzgsMi41N2ExLjYyLDEuNjIsMCwwLDEsLjM0LjQ3TDI4LjQsNjAuNmMuMTcuMjguMzYuNTUuNTUuODJsLS4xMi4xWiIvPjxwYXRoIGQ9Ik0zMi44Niw0NS43NWMtLjE3LTMuMjQtLjM0LTYuODktLjUyLTEwLjEuMzMuMTkuNTkuMzcuODYuNTFsNi43MiwzLjQ4YTcsNywwLDAsMSwuNjMuMzdzLS41NC4zMS0uODQuNTFjLS44OC41Ny0xLjc4LDEuMTMtMi42NSwxLjczbC0uNzYuNTNjLS42LjQ1LTEuMTkuOTItMS43NSwxLjQyUzMzLjQzLDQ1LjI0LDMyLjg2LDQ1Ljc1WiIvPjxwYXRoIGQ9Ik0xOS4yNyw0MiwxOCw0Ni42NSw5LjMxLDQwLjc0bDAtLjE5WiIvPjxwYXRoIGQ9Ik0zMywyMS43NWMtLjMtLjA4LTguODItMi44My05LjA3LTNzLS4xMi0uMywwLS40NWExLjM1LDEuMzUsMCwwLDEsLjQ3LS4zNSw1LjI5LDUuMjksMCwwLDEsMS4zMi0uMzFjLjMxLDAsLjYyLS4wOC45NC0uMTEsMS4wNi0uMTEsMi4xMy0uMTIsMy4yLS4xNGExLjQ0LDEuNDQsMCwwLDEsMS4zNywxQzMxLjczLDE5LjMzLDMyLjkxLDIxLjQ4LDMzLDIxLjc1WiIvPjxwYXRoIGQ9Ik0zNy4yNCwxOUMzNC45MSwyMS4xNywzNCwyMS45LDM0LDIxLjljLS42NS0xLjIxLTEuMjgtMi4zNC0xLjg5LTMuNDktLjE3LS4zMy4xNS0uNzMuNjYtLjc4YTMsMywwLDAsMSwuNzguMDdsMi40OS40QzM2LjU0LDE4LjIsMzcuODUsMTguNDUsMzcuMjQsMTlaIi8+PHBhdGggZD0iTTM5LjcsMjQuNDRhLjE1LjE1LDAsMCwxLDAsLjA5YzAsLjQyLS43Ni41Ny0xLjA2LjU5LS43Mi4wNi0xLjQ0LjA5LTIuMTYuMXMtMi4wOS4yOC0yLjc0LS40M2ExLjI5LDEuMjksMCwwLDEtLjIyLTEsNy40MSw3LjQxLDAsMCwxLC4xMS0xYzEuMS0uNTIsMi4xNC4xMiwzLjE5LjUxbDEuNzUuNjZDMzguNzgsMjQsMzkuNTgsMjQuMTcsMzkuNywyNC40NFoiLz48L3N2Zz4=
```

Comment:
What is this image?

Reply:
it is the logo, YML download from XSOAR had this